common/app/layout/Front.scala

```diff
@@ -369,22 +369,52 @@ object Front extends implicits.Collections {
   def deduplicate(
     seen: Set[TrailUrl],
     container: Container,
-    faciaContentList: Seq[FaciaContent]
-    ): (Set[TrailUrl], Seq[FaciaContent], (Seq[DedupedItem], Seq[DedupedItem])) = {
+    faciaContentList: Seq[FaciaContent],
+    isNetworkFront: Boolean,
+    isEditorialFront: Boolean
+    )(implicit request: RequestHeader): (Set[TrailUrl], Seq[FaciaContent], (Seq[DedupedItem], Seq[DedupedItem])) = {
```

Comment:
What's using this `implicit request`? Have you thought about trying to not carry it around?